src/query/config/src/outer_v0.rs

```diff
@@ -573,6 +584,85 @@ impl TryInto<InnerStorageHdfsConfig> for HdfsConfig {
     }
 }
 
+#[derive(Clone, PartialEq, Eq, Serialize, Deserialize, Args)]
+#[serde(default)]
+pub struct ObsStorageConfig {
+    /// Access key for OBS storage
+    #[clap(long = "storage-obs-access-key-id", default_value_t)]
+    pub access_key_id: String,
+
+    /// Secret key for OBS storage
+    #[clap(long = "storage-obs-secret-access-key", default_value_t)]
+    pub secret_access_key: String,
+
+    /// Bucket for OBS
+    #[clap(long = "storage-obs-bucket", default_value_t)]
+    pub bucket: String,
+
+    /// Endpoint URL for OBS
+    ///
+    /// # TODO(xuanwo)
+    ///
+    /// Clap doesn't allow us to use endpoint_url directly.
+    #[clap(long = "storage-obs-endpoint-url", default_value_t)]
+    #[serde(rename = "endpoint_url")]
+    pub endpoint_url: String,
```

Comment:
Due to the limitation of `clap`, we can't use the same field with other structs. Please add an `obs_` prefix for them.

Reply:
fixed